app/src/main/java/com/stevesoltys/seedvault/ui/storage/StorageActivity.kt

```diff
@@ -35,7 +35,9 @@ class StorageActivity : BackupActivity() {
             val authority = uri.authority ?: throw AssertionError("No authority in $uri")
             val storageRoot = StorageRootResolver.getStorageRoots(this, authority).getOrNull(0)
             if (storageRoot == null) {
-                viewModel.onUriPermissionResultReceived(null)
+                viewModel.onSafOptionChosen(
+                    StorageRootResolver.getFakeStorageRootForUri(this, uri))
+                viewModel.onUriPermissionResultReceived(uri)
```

Comment:
Oops, I guess that must have broken in some recent refactoring.

Btw. I don't really like this code in the activity. Maybe we should move it into the ViewModel and start (unit-) testing critical code paths like this.